core/src/server/rpc_module.rs

```diff
@@ -219,6 +245,16 @@ impl Methods {
 		self.callbacks.get_key_value(method_name).map(|(k, v)| (*k, v))
 	}
 
+	/// Returns the kind of the method callback,
+	pub fn method_kind(&self, method_name: &str) -> MethodKind {
+		match self.method(method_name) {
```

Comment:
Instead of `MethodKind::Unknown`, wouyld it make more sense to return `Option<MethodKind>` here? (or alternately, rename it to somehting clearer eg `MethodKind::NotFound`?)

Reply:
ah, this code is not used so I removed it but I guess your comment still applies.

Lemme fix it `NotFound` is better